jib-core/src/main/java/com/google/cloud/tools/jib/blob/Blob.java

```diff
@@ -30,4 +30,12 @@
    * @throws IOException if writing the BLOB fails
    */
   BlobDescriptor writeTo(OutputStream outputStream) throws IOException;
+
+  /**
+   * Enables to notify to the HTTP client if the underlying request can be retried.
+   * @return {@code true} if {@link #writeTo(OutputStream)} can be called multiple times, {@code false} otherwise.
+   */
+  default boolean isRetryable() {
```

Comment:
I guess this respects it even if the comment is focused on http since it is the main usage today. The return part is very explicit about the fact it is about calling N > 1 times writeTo, no?

Reply:
Please remove `default`.